rmgpy/exceptions.py

```diff
@@ -0,0 +1,250 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+################################################################################
+#
+#   RMG - Reaction Mechanism Generator
+#
+#   Copyright (c) 2002-2017 Prof. William H. Green (whgreen@mit.edu), 
+#   Prof. Richard H. West (r.west@neu.edu) and the RMG Team (rmg_dev@mit.edu)
+#
+#   Permission is hereby granted, free of charge, to any person obtaining a
+#   copy of this software and associated documentation files (the 'Software'),
+#   to deal in the Software without restriction, including without limitation
+#   the rights to use, copy, modify, merge, publish, distribute, sublicense,
+#   and/or sell copies of the Software, and to permit persons to whom the
+#   Software is furnished to do so, subject to the following conditions:
+#
+#   The above copyright notice and this permission notice shall be included in
+#   all copies or substantial portions of the Software.
+#
+#   THE SOFTWARE IS PROVIDED 'AS IS', WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+#   IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+#   FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+#   AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+#   LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+#   FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+#   DEALINGS IN THE SOFTWARE.
+#
+################################################################################
+
+"""
+This module contains classes which extend Exception for usage in the RMG module
+"""
+
+class ActionError(Exception):
+    """
+    An exception class for errors that occur while applying reaction recipe
+    actions. Pass a string describing the circumstances that caused the
+    exceptional behavior.
+    """
+    pass
+
+class AtomTypeError(Exception):
+    """
+    An exception to be raised when an error occurs while working with atom
+    types. Pass a string describing the circumstances that caused the
+    exceptional behavior.
+    """
+    pass
+
+class ChemicallySignificantEigenvaluesError(Exception):
+    """
+    An exception raised when the reservoir state method is unsuccessful for
```

Comment:
shouldn't this be the "chemically significant eigenvalue method" rather than the "reservoir state method"?